cdap-common/src/main/java/io/cdap/cdap/common/http/AuthenticationChannelHandler.java

```diff
@@ -53,6 +53,8 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
       currentUserId = request.headers().get(Constants.Security.Headers.USER_ID);
       currentUserIP = request.headers().get(Constants.Security.Headers.USER_IP);
       String authHeader = request.headers().get(HttpHeaderNames.AUTHORIZATION);
+      LOG.trace("Got user ID '{}', user IP '{}', and authorization header length '{}'", currentUserId, currentUserIP,
+                authHeader == null ? "NULL" : String.valueOf(authHeader.length()));
```

Comment:
Do we need to log the auth header? I am a bit concerned logging that.

Reply:
It's only length, not the header itself